api/descriptions/settings/registry_post.md

```diff
@@ -45,3 +48,41 @@ curl -k \
   } ' \
   'https://<CONSOLE>/api/v<VERSION>/settings/registry'
 ```
+
+Starting with 30.03, you can directly add a GitLab Container Registry.
+To add settings for a GitLab Container Registry, you must specify the following parameters:
+
+* **version**:  Specify the value *gitlab* for GitLab Container Registry.
+* **registry**: Specify the GitLab registry URL address. Example, for native registries, you can specify the address as "https://registry.gitlab.com" 
+* **credentialID**: Specify the GitLab credential that you added in the credential store in Prisma Cloud Compute. For example, an API token that has atleast the *read_api* scope.
+* **gitlabRegistrySpec**: Specify at least one of the following fields:
+    * **userID**: Specify your GitLab user ID to add all registries associated with it.
+    * **projectIDs**: Specify the project IDs to add all registries associated with a GitLab project.
+    * **groupIDs**: Specify the group ID to add all registries associated with a GitLab group.
+    * **excludedGroupIDs**: Specify the top-level group IDs that you don't want to add.
+
+Refer to the following example cURL command:
+
+```bash
+curl -k \
+  -u <USER> \
+  -H 'Content-Type: application/json' \
+  -X POST \
+  -d '
+  {
+    "version":"gitlab",
+    "registry":"https://registry.gitlab.com",
+    "namespace":"",
+    "repository":"",
+    "tag":"",
+    "credentialID":"<GitLab Token>",
```

Comment:
This is not correct: 
It should be the credential ID  of the credential the user created under credential store. Should be fixed before release